src/Runner/ErrorHandler.php

```diff
@@ -140,13 +152,13 @@ public function __invoke(int $errorNumber, string $errorString, string $errorFil
                     $suppressed,
                 );
 
-                break;
+                throw new ErrorException('Any E_*_ERROR must abort execution');
```

Comment:
better throwable class is probably wanted, also, it should extend `\Error` not `\Exception`

Reply:
@sebastianbergmann Thank you for merging this fix. I did not add a custom class as I do not know how to name it and where to put it, so I would be grateful if you can help me and add a custom PHPUnit class for this exception and base it from `\Error` so `catch (\Exception $e)` clauses do not catch/handle it. This should be true for all PHPUnit exceptions.